File: microservices/PassengerService/src/main/java/com/modsen/taxi/passengerservice/domain/Passenger.java
package com.modsen.taxi.passengerservice.domain;

import jakarta.persistence.*;
import lombok.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
@Entity
@Table(name = "passengers")
public class Passenger {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String firstName;

    private String lastName;

    private String email;

    private String phone;

    private Boolean isDeleted = false;
}

File: microservices/PassengerService/src/main/java/com/modsen/taxi/passengerservice/domain/Passenger.kt
package com.modsen.taxi.passengerservice.domain

import jakarta.persistence.*

@Entity
@Table(name = "passengers")
data class Passenger(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long? = null,

    var firstName: String,

    var lastName: String,

    var email: String,

    var phone: String,

    var isDeleted: Boolean = false
) {
    constructor() : this(null, "", "", "", "", false)
}
